cdc/sinkv2/eventsink/txn/mysql/mysql_test.go

```diff
@@ -663,18 +620,15 @@ func TestMysqlSinkNotRetryErrDupEntry(t *testing.T) {
 		return db, nil
 	}
 
-	sinkOpts := SinkOptionsDefault()
-	sinkOpts.getDBConn = mockDBInsertDupEntry
-	sinkOpts.dmlMaxRetry = 1
-
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 	changefeed := "test-changefeed"
 	sinkURI, err := url.Parse("mysql://127.0.0.1:4000/?time-zone=UTC&worker-count=1&safe-mode=false")
 	require.Nil(t, err)
-	sink, err := NewMySQLBackend(ctx, model.DefaultChangeFeedID(changefeed), sinkURI, sinkOpts)
+	sink, err := NewMySQLBackend(ctx, model.DefaultChangeFeedID(changefeed), sinkURI,
+		config.GetDefaultReplicaConfig(), mockDBInsertDupEntry)
```

Comment:
`dmlMaxRetry` is customed above.

Reply:
I moved it to bottom. Because it only for test, so I think it's ok. We have no way to custom it before we create the sink.